src/Shared/Constants.cs

```diff
@@ -77,6 +77,11 @@ internal static class MSBuildConstants
         /// </summary>
         internal const string DefaultTargetsMarker = ".default";
 
+        /// <summary>
+        /// Framework version against which our test projects should be built.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Framework version against which our test projects should be built.
        /// </summary>
        /// <remarks>
        /// The targeting pack for this version of .NET Framework must be installed
        /// on any machine that wants to run tests successfully, so this can be
        /// periodically updated.
        /// </remarks>
```